rustfmt.toml

```diff
@@ -0,0 +1 @@
+tab_spaces = 4
```

Comment:
I don't mind, just curious is there any specific reason to set this option explicitly to the default value?

Reply:
For some reason it kept formatting with 2 spaces for me, so I set this. Didn't realize it was a default. Maybe I have some global rustfmt file somewhere?

Reply:
Could be the case, rustfmt does support global config: https://github.com/rust-lang/rustfmt/blob/master/Configurations.md